src/pages/spawn/spawnHost/EditSpawnHostModal.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from "react";
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { Body } from "@leafygreen-ui/typography";
+import { Input, Select } from "antd";
+import isEqual from "lodash.isequal";
+import { Modal } from "components/Modal";
+import { InputLabel } from "components/styles";
+import {
+  Host,
+  InstanceTypesQuery,
+  InstanceTypesQueryVariables,
+  MyVolumesQuery,
+  MyHostsQueryVariables,
+} from "gql/generated/types";
+import { GET_INSTANCE_TYPES, GET_MY_VOLUMES } from "gql/queries";
+import {
+  HostExpirationField,
+  VolumesField,
+} from "pages/spawn/spawnHost/fields";
+
+const { Option } = Select;
+
+interface editSpawnHostState {
+  expiration: Date;
+  noExpiration: boolean;
+  displayName?: string;
```

Comment:
I think it could make sense to this required since displayName since the defualt value in defaultEditSpawnHostState is an empty string